rllib/policy/eager_tf_policy_v2.py

```diff
@@ -557,11 +561,29 @@ def compute_log_likelihoods(
             dist_inputs, self.dist_class, _ = self.action_distribution_fn(
                 self, self.model, input_batch, explore=False, is_training=False
             )
+            action_dist = self.dist_class(dist_inputs, self.model)
         # Default log-likelihood calculation.
         else:
-            dist_inputs, _ = self.model(input_batch, state_batches, seq_lens)
+            if self.config.get("_enable_rl_module_api", False):
+                if in_eval_mode:
```

Comment:
I think you should be explicit and name this flag explore=True/False?
you can use exploration in inference mode as well for example.

Reply:
This is not simply explore=True / False thing. It's wether I should use forward_train() or forward_explore(). explore=True/False sets the choice between forward_inference and forward_exploration.